testproject/Assets/Tests/Runtime/MultiprocessRuntime/NetworkVariablePerformanceTests.cs

```diff
@@ -0,0 +1,244 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using MLAPI.NetworkVariable;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+using UnityEngine;
+using UnityEngine.Profiling;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using static ExecuteStepInContext;
+using Object = UnityEngine.Object;
+
+namespace MLAPI.MultiprocessRuntimeTests
+{
+    public class NetworkVariablePerformanceTests : BaseMultiprocessTests
+    {
+        protected override int NbWorkers { get; } = 1;
+        private const int k_MaxObjectsToSpawn = 10000;
+        private List<OneNetVar> m_ServerSpawnedObjects = new List<OneNetVar>();
+        private static GameObjectPool<OneNetVar> s_ServerObjectPool;
+        private CustomPrefabSpawnerForPerformanceTests<OneNetVar> m_ClientPrefabHandler;
+        private OneNetVar m_PrefabToSpawn;
+        protected override bool m_IsPerformanceTest => true;
+
+        private class OneNetVar : NetworkBehaviour
+        {
+            public static int nbInstances;
+            public NetworkVariableInt oneInt = new NetworkVariableInt();
+
+            public void Init()
+            {
+                nbInstances++;
+                if (IsServer)
+                {
+                    oneInt.Value = 1;
+                }
+            }
+
+            public static void Stop()
+            {
+                nbInstances--;
+            }
+        }
+
+        [OneTimeSetUp]
+        public override void SetupTestSuite()
+        {
+            base.SetupTestSuite();
+            SceneManager.sceneLoaded += OnSceneLoadedInitSetupSuite;
+        }
+
+        private void OnSceneLoadedInitSetupSuite(Scene scene, LoadSceneMode loadSceneMode)
+        {
+            InitPrefab();
+            s_ServerObjectPool = new GameObjectPool<OneNetVar>();
+            s_ServerObjectPool.Init(k_MaxObjectsToSpawn, m_PrefabToSpawn);
+        }
+
+        private void InitPrefab()
+        {
+            if (m_PrefabToSpawn == null)
+        {
+            var prefabCopy = Object.Instantiate(PrefabReference.Instance.referencedPrefab);
+            m_PrefabToSpawn = prefabCopy.AddComponent<OneNetVar>();
+        }
+        }
+
+        [UnityTest, Performance, MultiprocessContextBasedTest]
+        public IEnumerator TestSpawningManyObjects([Values(1, 2, 1000, 2000, 10000)] int nbObjects)
+        {
+            InitContextSteps();
+
+            // if (!TestCoordinator.Instance.isRegistering)
```

Comment:
Not a huuuuuge fan of this, is there a way to pass in a bool when we want to do this or a different attribute or something?

Just so we don't have any commented code running around.

Reply:
added that info to the buildinfo file